src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -86,22 +84,102 @@ it on a `<canvas>`:
       })
       .catch(error => console.error('grabFrame() error:', error));
 
+
+## Capabilities and settings
+
+There are a number of ways to manipulate the capture settings, depending on
+whether the changes would be reflected in the `MediaStreamTrack` or can only be
+seen after `takePhoto()`. For example, a change in `zoom` level is immediately
+propagated to the `MediaStreamTrack` whereas the red eye reduction, when set, is
+only applied when the photo is being taken.
+
+"Live" camera capabilities and settings are manipulated via the preview
+`MediaStreamTrack`: `MediaStreamTrack.getCapabilities()` returns a
+[`MediaTrackCapabilities`](https://w3c.github.io/mediacapture-image/#mediatrackcapabilities-section)
+dictionary with the concrete supported capabilities and the ranges or allowed
+values, e.g. supported zoom range or allowed white balance modes.
+Correspondingly, `MediaStreamTrack.getSettings()` returns a
+[`MediaTrackSettings`](https://w3c.github.io/mediacapture-image/#mediatracksettings-section)
+with the concrete current settings. Zoom, brightness, and torch mode belong to
+this category, for example:
+
+    var zoomSlider = document.querySelector('input[type=range]');
+    // ...
+    const capabilities = mediaStreamTrack.getCapabilities();
+    const settings = mediaStreamTrack.getSettings();
+    if (capabilities.zoom) {
+      zoomSlider.min = capabilities.zoom.min;
+      zoomSlider.max = capabilities.zoom.max;
+      zoomSlider.step = capabilities.zoom.step;
+      zoomSlider.value = settings.zoom;
+    }
+
+Note: `MediaStreamTrack.getCapabilities()` and `.getSettings()` are synchronous
+but the capabilities are only available when the camera starts actually
+streaming, so make sure there is a delay between calling these methods and
+`getUserMedia()`, see [crbug.com/711524](https://crbug.com/711524).
```

Comment:
@beaufortfrancois this note will stand out in the article (as it does e.g. in https://developers.google.com/web/updates/2016/12/imagecapture); I'm not sure how to make the necessary timeout between `getUserMedia()` and `getCapabilities()/getSettings()` more evident than this in the current snippets.  I will add the timeout and a link to the bug in https://github.com/GoogleChrome/samples/pull/503.